src/sentry/integrations/web/integration_extension_configuration.py

```diff
@@ -81,7 +93,9 @@ def get(self, request: HttpRequest, *args, **kwargs) -> HttpResponseBase:
                 )
                 if org_member and "org:integrations" in org_member.scopes:
                     try:
-                        pipeline = self.init_pipeline(request, organization, request.GET.dict())
+                        pipeline = self.init_pipeline(
+                            Request(request=request), organization, request.GET.dict()
```

Comment:
I'm not sure this is going to be safe. Perhaps we an expand the types on the relevant `Pipeline` logic instead of attempting to convert the request types.

Reply:
fwiw it's very likely the type for `init_pipeline` is already wrong -- a good percentage of `request: Request` annotations were done sweepingly before mypy was set up and they're almost all wrong (when most of the time they should be `request: HttpRequest`)